sw/device/sca/BUILD

```
@@ -37,3 +37,47 @@ opentitan_flash_binary(
         "//sw/device/sca/lib:simple_serial",
     ],
 )
+
+opentitan_flash_binary(
```

It looks like the `_serial.c` files are set up as tests; in this case, you should be able to use the `opentitan_functest` rule so that they work with `bazel test`:
```suggestion
opentitan_functest(
```

---

@jadephilipoom, this is a good suggestion for consistency. As I am new and this is my first PR, I have a few questions to understand the infrastructure better:

- In our side-channel drivers (`*_serial.c`), the code does a few hardware initializations (setting up UARTs, GPIOs, etc.), and then. it goes into an infinite loop and waits for the [simpleserial commands](https://wiki.newae.com/SimpleSerial) (`set_key`, `encrypt`, etc.) coming through the UART interface. In our experiments we send the simpleserial commands from Python scripts in [ot-sca repository](https://github.com/lowRISC/ot-sca/blob/master/cw/capture.py). 

  As the `*_serial.c` code will be in an infinite loop, they will probably not send any status back to the `bazel test`. Would it cause any problems regarding CI?

- We may also need to apply the same change for the other targets in the BUILD file (`aes_serial.c`, `sha3_serial.c`) for consistency. 

@jadephilipoom, @vogelpi  Considering these, what would be a good approach here? I can either apply the change if it does not cause any problems regarding CI test statistics or I can note it as an item in #13905.

---

>   As the `*_serial.c` code will be in an infinite loop, they will probably not send any status back to the `bazel test`. Would it cause any problems regarding CI?

Yes, since the CI automatically picks up tests it would spin on an infinite-loop test; best to leave it as `flash_binary` then!